api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -12,7 +12,7 @@ import "envoy/api/v2/core/socket_option.proto";
 import "envoy/api/v2/listener.proto";
 import "envoy/config/metrics/v2/stats.proto";
 import "envoy/config/overload/v2alpha/overload.proto";
-import "envoy/config/trace/v2/trace.proto";
```

Comment:
I'm of mixed mind on making changes to v2 here. It's not a breaking API change to move messages between files in the same package with `import public`, but prefer not modifying frozen protos. Knowing the context of https://github.com/envoyproxy/envoy/issues/10576 though, I think they wanted this fixed for gRPC in the v2 xDS APIs, so willing to make an exception as this bordered the v2/v3 migration.

Reply:
Yes, I'm applying the changes to `v2` as well because that is the api version mentioned in #10576  (see the attached error message)